modules/nodes-pods-autoscaling-custom-rn.adoc

```diff
@@ -26,23 +26,37 @@ The following table defines the Custom Metrics Autoscaler Operator versions for
 |{product-title} version
 |General availability
 
-|2.8.2
-|4.13
-|Technology Preview
-
-|2.8.2
+|2.8.2-174
 |4.12
 |Technology Preview
 
-|2.8.2
+|2.8.2-174
 |4.11
 |Technology Preview
 
-|2.8.2
+|2.8.2-174
 |4.10
 |Technology Preview
 |===
 
+[id="nodes-pods-autoscaling-custom-rn-282-174_{context}"]
+== Custom Metrics Autoscaler Operator 2.8.2-174 release notes
+
+This release of the Custom Metrics Autoscaler Operator 2.8.2-174 provides new features and bug fixes for running the Operator in an {product-title} cluster. The components of the Custom Metrics Autoscaler Operator 2.8.2-174 were released in link:https://access.redhat.com/errata/RHEA-2023:1683[RHEA-2023:1683].
+
+[IMPORTANT]
+====
+The Custom Metrics Autoscaler Operator is currently a link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview] feature.
+====
+
+[id="nodes-pods-autoscaling-custom-rn-282-174-new_{context}"]
+== New features and enhancements
+
+[id="autoscaling-custom-2-8-2-must-gather"]
+=== must-gather support
+
+You can now collect data about the Custom Metrics Autoscaler Operator and its components by using the {product-title} must-gather tool. For more information, see xref:../../nodes/pods/nodes-pods-autoscaling-custom.html#nodes-pods-autoscaling-custom-gather_nodes-pods-autoscaling-custom[Gathering debugging data].
```

Comment:
```suggestion
You can now collect data about the Custom Metrics Autoscaler Operator and its components by using the {product-title} `must-gather` tool. For more information, see xref:../../nodes/pods/nodes-pods-autoscaling-custom.html#nodes-pods-autoscaling-custom-gather_nodes-pods-autoscaling-custom[Gathering debugging data].
```

Reply:
@bscott-rh Which is the _official_ way to present the `must-gather`/must-gather tool? I see it both ways. I prefer the latter. 
Like:

> Use the `oc adm must-gather` command to launch the {product-title} must-gather tool.

Reply:
I think that `must-gather` is the proper way because it's actually a binary/command that customers would use in the same way as `oc`. I based my comment off of the content in your xref which uses monospace.